input.go

```diff
@@ -234,6 +234,18 @@ func IsStandardGamepadLayoutAvailable(id GamepadID) bool {
 	return uiDriver().Input().IsStandardGamepadLayoutAvailable(id)
 }
 
+// UpdateStandardGamepadLayoutMappings updates the gamepad layout definitions using a set of definitions in SDL_GameControllerDB format.
+//
+// Returns whether the mappings were applied, and whether an error occurred parsing the mappings.
```

Comment:
* `This reports whether ...` or `UpdateStandardGamepadLayoutMappings reports whether ...`
* `and this returns an error occurred in parsing the mappings if it exists`

Reply:
First applied. Sorry, I do not understand your second suggestion; rephrased another way, is this clearer?

Reply:
The latest version seems good, thanks!